contrib/pcre/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "pcre"
```

Comment:
we definitely don't want to bring back this one (it was previously removed, is long unmaintained/obsolete, and contains security issues)

upstreams should migrate to pcre2

Reply:
I agree.  There is a PR on snort3 project from 2022

https://github.com/snort3/snort3/issues/254